src/__spec_helper__/test-utils.ts

```diff
@@ -226,6 +231,14 @@ const backgroundProps = [
   ["backgroundRepeat", "background-repeat", "no-repeat"],
 ] as const;
 
+const positionProps = [
+  ["top", "0px"],
+  ["bottom", "0px"],
+  ["right", "0px"],
+  ["left", "0px"],
+  ["position", "fixed"],
```

Comment:
are there not other position values to test as well (sticky etc)?

Reply:
Yeah good point. I cover it in the cypress tests. I wrote this test before the cypress one, where I actually looked for all the values. I'll get them added in 😄